client/src/components/AddNewProfile.js

```diff
@@ -127,10 +157,35 @@ const AddNewProfile = () => {
           onChange={handleChange}
         />
       </Form.Field>
+      <Form.Field>
+        <Dropdown
+          onChange={profileTypeChange}
+          placeholder="Select profile type"
+          fluid
+          selection
+          options={profileTypeOptions}
+        />
+      </Form.Field>
+      <Form.Field>
+        <Dropdown
+          onChange={statusChange}
+          placeholder="Select status"
+          defaultValue="new"
+          fluid
+          selection
+          options={statusOptions}
+        />
+      </Form.Field>
+      <Form.Field>
+        <label>Join date</label>
+        <SemanticDatepicker onChange={dateChange} />
```

Comment:
```suggestion
        <label for="join-date">Join date</label>
        <SemanticDatepicker  id="join-date" onChange={dateChange} />
```
Good job adding a label here. However, all labels need to be linked to inputs to make them accessible to screen readers. We do this by adding a `for` to the label which points to the `id` of the input.

Unfortunately, we can't do this with the two `Dropdown`s above because the `Dropdown` component doesn't take an `id` prop and put it on the input so we have to leave the `Dropdown` labels unlinked for now.

https://blog.prototypr.io/five-minute-accessibility-input-and-labels-fc88990b7d47